src/containers/info.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import NavBar from './../components/nav_bar.jsx';
+
+const Info = () => (
```

Comment:
this is not a container this is functional component. move the file to the components folder